treeherder/model/management/commands/cycle_data.py

```diff
@@ -62,6 +63,11 @@ def handle(self, *args, **options):
                                                 options['sleep_time'])
             self.debug("Deleted {} jobs from {}".format(rs_deleted,
                                                         repository.name))
+            if repository.expire_performance_data:
```

Comment:
So this cycles through the repository table, where expire_performance_data is True by default for new records. Question, where do you actually check the 'expire_performance_data' value in the fixtures/repository.json? (Where does that json setting override what's in the table by default)?